src/Hotel.js

```diff
@@ -38,15 +41,23 @@ class Hotel {
     return availableRoomsOnDate;
   }
 
-  bookRoomForGuest(userID, date, roomNumber) { 
-    let bookingData =  { userID: userID, date: date, roomNumber: roomNumber }
-    // call fetchPOST here, I think
-    return bookingData;
+  bookRoomForGuest(userID, date, roomNum) { 
+    let bookingData =  { "userID": userID, "date": date, "roomNumber": roomNum }
+    console.log('bookingData', typeof bookingData.userID)
+    apiRequests.postData('bookings/bookings', bookingData)
+      .then(() => {
+        this.updateBookingsData();
+      })
   }
 
-
-
-     /*
+  updateBookingsData() {
```

Comment:
So glad to get this figured out. And! I think I can also use it for updating after a DELETE!